src/reanimated2/js-reanimated/global.ts

```diff
@@ -1,38 +1,40 @@
 // In order to keep bundle size down, we treat this file as a polyfill for Web.
 
 import { shouldBeUseWeb } from '../PlatformChecker';
-
-if (shouldBeUseWeb()) {
-  global._frameTimestamp = null;
-  global._setGlobalConsole = (_val) => {
-    // noop
-  };
-  global._measure = () => {
-    console.warn(
-      "[Reanimated] You can't use `measure` with Chrome Debugger or with web version"
-    );
-    return {
-      x: 0,
-      y: 0,
-      width: 0,
-      height: 0,
-      pageX: 0,
-      pageY: 0,
+export default (() => {
```

Comment:
Can we name this lambda `initializeGlobalsForWeb` and call it as
```ts
export default initializeGlobalsForWeb();
```
? Also, a comment here why we need to do it would be valuable.